### ISSUE TITLE
Confusing installation?

### ISSUE THREAD

**Issue by jadasse** [ttalvitie/browservice#8]
I'm not too experienced with compiling, have you thought of making installation executables?

**Comment by m1k1o** [comment]
Or maybe Dockerfile would be nice to have.

**Comment by jadasse** [comment]
Is dockerfile a simplified compiler?

**Comment by m1k1o** [comment]
Kind of, image can be built using Dockerfile and then run in a simple manner. https://docs.docker.com/engine/reference/builder/

**Comment by jadasse** [comment]
Cool, I'll check it out

**Comment by Dylan1496** [comment]
@jadasse I did make an AUR package for this to automate the install. Unfortunately, this only works on Arch and derivatives. The pkgbuild file is here: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=browservice

**Comment by ttalvitie** [comment]
Having prebuilt binaries would be nice indeed, however, setting up and maintaining builds for four architectures and many Linux distributions that are most likely binary incompatible takes some effort which I haven't had time to put in. Ideally, someone would package Browservice into the package managers of the Linux distributions; the AUR package by @Dylan1496 is a good start in this direction.

Meanwhile, we could try to make the manual setup easier. @jadasse, which part of the current setup procedure is the most confusing one? The idea was that you wouldn't need any experience with compiling, as you should simply copy the required commands from the README.

**Comment by jadasse** [comment]
Oh, I meant for Windows 95 internet explorer 5.5

**Comment by ttalvitie** [comment]
@Dylan1496 thank you for creating the AUR package. One issue some people have had with it is that it seems to compile all the modules in parallel, which uses too much memory on some systems. This is because `make` is invoked with the flag `-j` without a limit; you probably should limit the number of parallel jobs to the number of cores by using `make -j$(nproc)` instead.

**Comment by jadasse** [comment]
Where should I put the commands?

**Comment by ttalvitie** [comment]
@jadasse you don't compile or run Browservice on the Windows 95 machine; it must run on a separate modern Linux machine. The Windows 95 machine accesses the Browservice instance running on the Linux machine over a local area network. The [YouTube video on Browservice by Michael MJD](https://www.youtube.com/watch?v=5MjZdKtv9ak) does a good job of explaining the setup.

**Comment by jadasse** [comment]
I wasnt going to compile it on a windows 95 virtual machine, i was trying to compile it on my windows 10 so i can put it in an iso and insert it in the vm to test it out on ie 5.5

**Comment by ttalvitie** [comment]
> I wasnt going to compile it on a windows 95 virtual machine, i was trying to compile it on my windows 10 so i can put it in an iso and insert it in the vm to test it out on ie 5.5

Make sure that you read the README (and watch the video) carefully to understand the structure of the Browservice setup. No part of the compiled Browservice is directly installed on the Windows 95 machine; the Windows 95 machine only needs IE5.5 and a network connection to the Linux machine running Browservice. Browservice does not run on Windows 10, so you need either a Linux VM or Windows Subsystem for Linux.

**Comment by jadasse** [comment]
Oh. I'll have to close this because this isn't windows compatible, I'm trying to find a way to browse modern internet (e.g. watch YouTube on Windows 9x) on old oses with windows compatibility.
